File: TP/TP1/Code/4E.cpp
/* Transformer un entier positif en un nombre de semaines et de jours */
#include <iostream>

int main()
{
  int a;

  std::cout << "Donne moi un chiffre : " << std::flush;
  std::cin >> a;

  std::cout << a << " => " << a / 7 << " semaines et " 
            << a % 7 << " jours." << std::endl;
  
  return 0;
}

File: TP/TP1/Code/Python/4E.py
#!/usr/bin/python
# Transformer un entier positif en un nombre de semaines et de jours
print('Donne moi une valeur : ', end='')
foo = int(input())

# I ❤ Python
print(foo, 'jours c\'est', foo // 7, 'semaine(s) et', foo % 7, 'jours')
